bounties/packagist/dolibarr/dolibarr/4/vulnerability.json

```diff
@@ -0,0 +1,55 @@
+{
+  "PackageVulnerabilityID": "3",
+  "DisclosureDate": "24/09/2020",
+  "AffectedVersionRange": "Dolibarr 13.0.0-alpha",
+  "Summary": "Username Enumeration on Password Reset",
+  "Contributor": {
+    "Discloser": "kajalNair",
+    "Fixer": ""
+  },
+  "Package": {
+    "Registry": "packagist",
+    "Name": "dolibarr/dolibarr",
+    "URL": "https://packagist.org/packages/dolibarr/dolibarr"
+  },
+  "CWEs": [
+    {
+      "ID": "200",
+      "Description": "Exposure of Sensitive Information to an Unauthorized Actor
+"
```

Comment:
Just pop the speech marks back up to the previous line! 🍰

Reply:
done.
This has been already reviewed by @Mik317. A new PR is created against the closed one.
Thanks! 👍